contrib/lang/python/packages.el

```diff
@@ -172,7 +172,10 @@ which require an initialization must be listed explicitly in the list.")
                        'python-setup-shell))
     :config
     (progn
-      (add-hook 'inferior-python-mode-hook 'smartparens-mode)
+      (add-hook 'inferior-python-mode-hook '(lambda ()
+                                              (smartparens-mode)
+                                              (evil-search-highlight-persist -1)
+                                              (global-evil-search-highlight-persist -1)))
```

Comment:
I disabled globally just because otherwise `evil-search-highlight-persist` still takes in effect.

If `evil-search-highlight-persist` is not enabled globally by default, then there's no need to disable it =), I took it out.